### ISSUE TITLE
Error while importing the library in Ipython

### ISSUE THREAD

**Issue by puneetjindal** [erinhengel/Textatistic#1]
Hi,
I am getting this error on windows 10.
**IOError: Cannot load hyphen dictionary.**

**Comment by erinhengel** [comment]
Hi,

take a look at the documentation (http://www.erinhengel.com/software/textatistic/) under Installation:

Textatistic uses the PyHyphen hyphenation library which is not included in Python's original installation. Although pip installs PyHyphen automatically with Textatistic, it may omit the actual dictionaries. If your Textatistic installation was successful but you get an error that no dictionaries are installed, either uninstall PyHyphen and then reinstall it from source or manually install the dictionaries yourself (instructions).

If you have any idea how to fix this error, I'd greatly appreciate it!

**Comment by ccassion** [comment]
Hello,

Has this issue been resolved by either party?

**Comment by erinhengel** [comment]
No, you still need to install PyHyphen's dictionaries manually.

**Comment by JustinBirchard** [comment]
Hello, I am new to Python and GitHub. Hoping this might be helpful to others--

I was trying to install Textatistic and ran into a similar error caused by PyHyphen. However, my problem was not with the dictionaries as mentioned by others, instead I got a warning saying: 
_"Microsoft C++ 14.0 Build Tools Required , download available at visualstudio.microsoft.com/downloads/"_

**However, the url listed in the error message is outdated and incorrect** the correct URL to download is:
https://visualstudio.microsoft.com/visual-cpp-build-tools/

It is a very large download (4gb) and the installation takes about 5 minutes. 

After installing Microsoft C++ Build Tools Textatistic and PyHyphen seem to be working fine. I am using Anaconda and I tested in ipython and jupyter labs with no issues so far.

Thank you Erin for the cool library!
